pkg/vulnsrc/osv/types.go

```diff
@@ -1,12 +1,80 @@
 package osv
 
-import "golang.org/x/vuln/osv"
+import (
+	"time"
+)
 
-// source: https://github.com/golang/vuln/blob/9d39a965865fd1d0030df18602433a01f679fd7d/osv/json.go
+type RangeType string
+
+const RangeTypeGit RangeType = "GIT"
+
+type Ecosystem string
+
+// Module identifies the Go module containing the vulnerability.
+// Note that this field is called "package" in the OSV specification.
+//
+// See https://ossf.github.io/osv-schema/#affectedpackage-field.
+type Module struct {
+	Path      string    `json:"name"`
+	Ecosystem Ecosystem `json:"ecosystem"`
+}
```

Comment:
`golang/vuln` uses `Go` specific field names ( i mean module name). But we don't get `Go` advisories from OSV.
May we want to use names from `OSV schema`. @knqyf263 wdyt?